python-checks/src/test/resources/checks/hotspots/httpOnlyCookie.py

```diff
@@ -73,13 +73,13 @@ def flask_tests():
     from flask import Response, make_response, redirect
 
     response1 = Response('OK')
-    response1.set_cookie('c1', 'value') # Noncompliant
+    response1.set_cookie('c1', 'value') # FN
```

Comment:
`set_cookie` is no longer resolved to be `flask.wrappers.Response.set_cookie` nor `werkzeug.wrappers.BaseResponse.set_cookie` but instead `werkzeug.sansio.response.Response.set_cookie`. It is accurate, as this is where the method is inherited from, but possibly problematic when trying to define sensitive methods.